packages/core/src/Breadcrumbs.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
Add to v2

Reply:
PR created! https://github.com/priceline/design-system/pull/677